example/anisotropic_diffusion.cpp

```diff
@@ -0,0 +1,99 @@
+#include <boost/gil/algorithm.hpp>
+#include <boost/gil/typedefs.hpp>
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/extension/io/png.hpp>
+#include <boost/gil/io/typedefs.hpp>
+#include <boost/gil/image_processing/numeric.hpp>
+
+#include <cmath>
+#include <string>
+#include <vector>
+#include <type_traits>
+
+namespace gil = boost::gil;
+
+#include <iostream>
+
+void gray_version(const std::string& input_path, const std::string& output_path, unsigned int iteration_count, unsigned int kappa)
```

Comment:
You're welcome to [Join the East Const revolution!](https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/) 😆

Reply:
Joined :-)